client/keys/utils.go

```diff
@@ -38,16 +38,7 @@ func NewInMemoryKeyBase() keys.Keybase { return keys.NewInMemory() }
 // NewKeyBaseFromHomeFlag initializes a keyring based on configuration. Keybase
 // options can be applied when generating this new Keybase.
 func NewKeyringFromHomeFlag(input io.Reader, opts ...keys.KeybaseOption) (keys.Keybase, error) {
```

Comment:
```suggestion
func NewKeyring(input io.Reader, opts ...keys.KeybaseOption) (keys.Keybase, error) {
```

Reply:
It's doing much more than just home flag. It's also doing from service name and from backend.

Reply:
I removed it